assets/src/components/utils/BindingInput.tsx

```diff
@@ -31,14 +31,26 @@ const FETCHER = {
 }
 
 export function fetchUsers(client, query, setSuggestions) {
-  client.query({ query: SEARCH_USERS, variables: { q: query, all: true } })
-    .then(({ data: { users: { edges } } }) => edges.map(({ node }) => ({ value: node, label: userSuggestion(node) })))
+  client
+    .query({ query: SEARCH_USERS, variables: { q: query, all: true } })
+    .then(({
+      data: {
+        users: { edges },
+      },
+    }) => edges.map(({ node }) => ({ value: node, label: userSuggestion(node) })))
     .then(setSuggestions)
 }
 
-export function fetchGroups(client, query, setSuggestions) {
-  client.query({ query: SEARCH_GROUPS, variables: { q: query } })
-    .then(({ data: { groups: { edges } } }) => edges.map(({ node }) => ({ value: node, label: groupSuggestion(node) })))
+export function fetchGroups(client: ApolloClient<any>, query, setSuggestions) {
+  client
+    .query<SearchGroupsQuery, SearchGroupsQueryVariables>({
+      query: SearchGroupsDocument,
+      variables: { q: query },
+    })
+    .then(({ data }) => data?.groups?.edges?.map(edge => ({
+      value: edge?.node,
+      label: groupSuggestion(edge?.node),
+    })))
```

Comment:
@dogmar On a side note, I like your code formatting but it looks like it's your own tool. Perhaps you could add it to our setup somehow if others agree? I mean it's a lot easier to read above code after your changes.

cc @pluralsh/frontend

Reply:
I've been using [prettier](https://prettier.io/) and I'd love to integrate it into our setup if everyone else likes it. It's very opinionated, but I love just not having to think about formatting at all and letting it do its thing.

Reply:
With @floreks we were using prettier in our previous projects. Mostly not directly but with [gts](https://github.com/google/gts). It can serve as a base config but we might prepare something on our end.

And yes, I'd love to have consistent formatting no matter who is doing the changes.